lib/src/clojure_lsp/queries.clj

```diff
@@ -334,13 +334,13 @@
           (get analysis filename))))
 
 (defmethod find-references :var-usages
```

Comment:
It's not obvious, but the queries namespace went through several changes to get to this point. Now it only deals with the derefed db, never the atom. This is as it should be—a namespace called "queries" should not mutate anything.

Reply:
sounds perfect